src/Avalonia.Visuals/Animation/Keyframes/TransformKeyFrames.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Avalonia.Collections;
+using System.ComponentModel;
+using Avalonia.Animation.Utils;
+using System.Reactive.Linq;
+using System.Linq;
+using Avalonia.Media;
+
+namespace Avalonia.Animation.Keyframes
+{
+    /// <summary>
+    /// Key frames that handles <see cref="double"/> properties.
```

Comment:
Should this be "that handles Transform properties."? or "that handles double properties on Transforms"?